1MI0800224/week2/week2_task2.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+bool IsSymbolAorD(const char symbol){
+    return (symbol=='a' || symbol == 'd');
+}
+
+int main(){
+    int counter=0;
+    ofstream file("file.txt");
+    if(!file.is_open()){
+        cout<<"Error";
+        return -1;
+    }
+
+    int lenght;
+    cin>>lenght;
+    char* text = new char[lenght];
+    for(int i=0; i<lenght-1; i++){

```

Comment:
[get](https://cplusplus.com/reference/istream/istream/get/) can be used with a second argument to indicate how many symbols are to be read, [getline](https://cplusplus.com/reference/istream/istream/getline/) too